main.go

```diff
@@ -154,3 +208,78 @@ func exitWithError(msg ...interface{}) {
 	fmt.Println(msg...)
 	os.Exit(1)
 }
+
+// generates relevant nethermind chain configuration files
+// logic ported over from audius-docker-compose https://github.com/AudiusProject/audius-docker-compose/blob/stage/audius-cli#L848
+func configureChainSpec() {
+	extraVanity := "0x22466c6578692069732061207468696e6722202d204166726900000000000000"
+	extraSeal := "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+
+	// gather env config
+	// discovery-provider/stage.env for example
+	networkEnvPath := fmt.Sprintf("discovery-provider/%s.env", network)
+	networkEnv := dockerExec("cat", networkEnvPath)
+	networkEnvMap, err := godotenv.Unmarshal(networkEnv)
+	if err != nil {
+		exitWithError("Error unmarshalling network env:", err)
+	}
+
+	signers := networkEnvMap["audius_genesis_signers"]
+	extraData := fmt.Sprintf("%s%s%s", extraVanity, signers, extraSeal)
+
+	specTemplatePath := fmt.Sprintf("discovery-provider/chain/%s_spec_template.json", network)
+	specInput := dockerExec("cat", specTemplatePath)
+	var specData map[string]interface{}
+	err = json.Unmarshal([]byte(specInput), &specData)
+	if err != nil {
+		exitWithError("Unmarshall error:", err)
+	}
+
+	networkId := specData["params"].(map[string]interface{})["networkID"].(string)
```

Comment:
the `.(...)` is telling the compiler what the type is, parsing raw json in golang without a struct is kind of gnarly